garde/Cargo.toml

```diff
@@ -25,11 +25,11 @@ full = [
 
 serde = ["dep:serde", "compact_str/serde", "smallvec/serde"]
```

Comment:
Please add a comment in both `Cargo.toml` files which points out that changes to features may require updating the list of checked features in `garde_derive/syntax.rs`.